src/Medusa.ts

```diff
@@ -28,69 +28,67 @@
+            configModule: {
+                monitoring: MonitoringOptions;
+            };
+        };
 
-		if (configModule.monitoring) {
-			await loadMonitoringModule(this.#express, configModule.monitoring);
-		}
+        await loadMonitoringModule(configModule, this.#express, configModule.monitoring);
 
-		await validatorsLoader(moduleComponentsOptions.get('validator') ?? []);
-		await overrideEntitiesLoader(moduleComponentsOptions.get('entity') ?? []);
-		await overrideRepositoriesLoader(moduleComponentsOptions.get('repository') ?? []);
-		await adminApiLoader(
-			this.#express,
-			moduleComponentsOptions.get('middleware') ?? [],
-			moduleComponentsOptions.get('router') ?? []
-		);
-		await storeApiLoader(
-			this.#express,
-			moduleComponentsOptions.get('middleware') ?? [],
-			moduleComponentsOptions.get('router') ?? []
-		);
-		await databaseLoader(
-			moduleComponentsOptions.get('entity') ?? [],
-			moduleComponentsOptions.get('repository') ?? []
-		);
-		await pluginsLoadersAndListeners(this.#express);
-		await servicesLoader(moduleComponentsOptions.get('service') ?? []);
+        await validatorsLoader(moduleComponentsOptions.get('validator') ?? []);
+        await overrideEntitiesLoader(moduleComponentsOptions.get('entity') ?? []);
+        await overrideRepositoriesLoader(moduleComponentsOptions.get('repository') ?? []);
+        await adminApiLoader(
+            this.#express,
+            moduleComponentsOptions.get('middleware') ?? [],
+            moduleComponentsOptions.get('router') ?? []
+        );
+        await storeApiLoader(
+            this.#express,
+            moduleComponentsOptions.get('middleware') ?? [],
+            moduleComponentsOptions.get('router') ?? []
+        );
+        await databaseLoader(
+            moduleComponentsOptions.get('entity') ?? [],
+            moduleComponentsOptions.get('repository') ?? []
+        );
+        await pluginsLoadersAndListeners(this.#express);
+        await servicesLoader(moduleComponentsOptions.get('service') ?? []);
 
-		const { container, dbConnection } = await loaders({
-			isTest: process.env.NODE_ENV === 'test',
-			directory: this.#rootDir,
-			expressApp: this.#express,
-		});
+        const { container, dbConnection } = await loaders({
+            isTest: process.env.NODE_ENV === 'test',
+            directory: this.#rootDir,
+            expressApp: this.#express,
+        });
 
-		await migrationsLoader(moduleComponentsOptions.get('migration') ?? [], dbConnection);
+        await migrationsLoader(moduleComponentsOptions.get('migration') ?? [], dbConnection);
 
-		const endPoints = getEndpoints(this.#express);
-		for (const endPoint of endPoints) {
-			endPoint.methods.map((method) => {
-				logger.push('Route Mapped {%s, %s}', endPoint.path, method);
-			});
-		}
-		logger.flush();
+        const endPoints = getEndpoints(this.#express);
+        for (const endPoint of endPoints) {
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `↹↹`](https://app.codacy.com/gh/adrien2p/medusa-extender/pullRequest?prid=9061797)
```suggestion
		for (const endPoint of endPoints) {
```